src/Controller/FrontendController/AbstractFrontendModuleController.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2018 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Controller\FrontendModule;
+
+use Contao\CoreBundle\Controller\AbstractFragmentController;
+use Contao\BackendTemplate;
+use Contao\ModuleModel;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+abstract class AbstractFrontendModuleController extends AbstractFragmentController
+{
+    /**
+     * @param Request     $request
+     * @param ModuleModel $module
+     * @param string      $section
+     *
+     * @return Response
+     */
+    public function __invoke(Request $request, ModuleModel $module, string $section)
+    {
+        if ($this->showBackendWildcard($request)) {
+            return $this->getBackendWildcard($module, $request);
+        }
+
+        $template = $this->createTemplate($module, 'mod_');
+
+        $template->inColumn = $section;
+
+        if (is_array($classes = $request->attributes->get('classes'))) {
+            $template->class .= ' ' . implode(' ', $classes);
+        }
+
+        return $this->getResponse($template, $module, $request);
+    }
+
+    /**
+     * @param Request $request
+     *
+     * @return bool
+     */
+    protected function showBackendWildcard(Request $request)
+    {
+        return $this->get('contao.routing.scope_matcher')->isBackendRequest($request);
+    }
+
+    /**
+     * @param ModuleModel $module
+     * @param Request     $request
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    protected function getBackendWildcard(ModuleModel $module, Request $request)
```

Comment:
This method should actually be also available for content elements, as some of them may display dynamic data as well.

Reply:
It should not be in the parent class for sure, because the generated URL is different for different types. Also, none of the core elements have wildcards. What's the reason not to show dynamic data?

Reply:
And what's the reason to show backend wildcard for frontend modules?

Reply:
¯\_(ツ)_/¯

I guess it's also a quick link to edit the module, which does not make sense for an element. There are content elements that change their appearance in the backend, but never to a wildcard. The YouTube element does not generate an iframe but a link to the video for example.